kratos/utilities/geometrical_projection_utilities.h

```diff
@@ -199,6 +198,35 @@ class GeometricalProjectionUtilities
         return norm_2(rPointProjected.Coordinates()-p_c);
     }
 
+
+    /**
+     * @brief Computes the minimal distance to a line
+     * @details Projects over a line and if the point porjected is inside the line that distance is taken into consideration, otherwise the minimal between the two points in the line is considered
```

Comment:
```suggestion
     * @details Projects over a line and if the point projected is inside the line that distance is taken into consideration, otherwise the minimal between the two points in the line is considered
```